pkg/scheduler/apis/config/types.go

```diff
@@ -131,3 +140,63 @@ type KubeSchedulerLeaderElectionConfiguration struct {
 	// LockObjectName defines the lock object name
 	LockObjectName string
 }
+
+// Plugins include multiple extension points. When specified, the list of plugins for
+// a particular extension point are the only ones enabled. If an extension point is
+// omitted from the config, then the default set of plugins is used for that extension point.
+// If no plugin should be invoked at an extension point, an empty list must be provided.
+type Plugins struct {
+	// QueueSort is a list of plugins that should be invoked when sorting pods in the scheduling queue.
+	QueueSort []Plugin
+
+	// PreFilter is a list of plugins that should be invoked at "PreFilter" extension point of the scheduling framework.
+	PreFilter []Plugin
+
+	// Filter is a list of plugins that should be invoked when filtering out nodes that cannot run the Pod.
+	Filter []Plugin
+
+	// PostFilter is a list of plugins that should be invoked when a list of nodes that passed the filtering phase.
```

Comment:
PostFilter is a list of plugins that are invoked after filtering out infeasible nodes.